pkgs/desktops/gnome-3/extensions/wsmatrix.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, glib }:
+
+stdenv.mkDerivation rec {
+  name = "gnome-shell-wsmatrix-${version}";
+  version = "9543a73df7bac9476f8adeba9d7a14fdacb57b7d";
```

Comment:
Can you specify a release tag from https://github.com/mzur/gnome-shell-wsmatrix/releases as the version?